samtools.1

```diff
@@ -1263,6 +1263,24 @@ with the header in
 This command is much faster than replacing the header with a
 BAM\(->SAM\(->BAM conversion.
 
+By default this command outputs the BAM or CRAM file to standard
+output (stdout), but for CRAM format files it has the option to
+perform an insitu edit, both reading and writing to the same file.  No
+validity checking is performed on the header, nor that it is suitable
+to use with the sequence data itself.
+
+.B OPTIONS:
+.RS
+.TP 8
+.B -P, --no-PG
+Do not generate an @PG header line.
+.TP 8
+.B -I, --insitu
+Perform the header edit insitu, if possible.  This only works on CRAM
```

Comment:
On Tue, Jun 23, 2015 at 08:10:26AM -0700, John Marshall wrote:
> `sed` and `perl` call this **in-place** editing, and use the options `-i[suffix]`/`--in-place[=suffix]`... but if you really really like _in situ_, I think it's two words :smile:

I was wary of using -i as I thought we use it elsewhere already to
indicate input files, but I'm wrong - it's just -o for output.  -i is
used in targetcut (inPen) and stats (insert-size), -I is stats (ID)
and mpileup (skip-indels).  So I'm happy to change that to --in-place
and -i.

James

-- 
James Bonfield (jkb@sanger.ac.uk) | Hora aderat briligi. Nunc et Slythia Tova
                                  | Plurima gyrabant gymbolitare vabo;
  A Staden Package developer:     | Et Borogovorum mimzebant undique formae,
https://sf.net/projects/staden/   | Momiferique omnes exgrabure Rathi. 


-- 
 The Wellcome Trust Sanger Institute is operated by Genome Research 
 Limited, a charity registered in England with number 1021457 and a 
 company registered in England with number 2742969, whose registered 
 office is 215 Euston Road, London, NW1 2BE.